src/main/cli/settings.c

```diff
@@ -160,7 +160,8 @@ const char * const lookupTableAccHardware[] = {
     "BMI270",
     "LSM6DSO",
     "LSM6DSV16X",
-    "VIRTUAL"
+    "IIM42653",
+    "VIRTUAL",
```

Comment:
the is no comma after VIRTUAL if it is intentionay last item - it gets highlighted in diff when someone tries to add item after it
```suggestion
    "VIRTUAL"
```

Reply:
> Version if VIRTUAL is not kept last

ahh sorry I thought you were suggesting adding the comma. Will update

Reply:
the comma would go after "IIM42653", even if it is last entry (line will not change when new entry is appended). VIRTUAL is special case, it is always last